scanner/config.yaml.sample

```diff
@@ -6,11 +6,13 @@ indexer:
     conn_string: "host=/var/run/postgresql"
     password_file: ""
   get_layer_timeout: 1m
+  repository_to_cpe_url:
```

Comment:
We can leave this blank as Claircore uses https://access.redhat.com/security/data/metrics as default